lib/graphql/transaction.ts

```diff
@@ -120,4 +140,28 @@ export class Transaction extends IterableModel<TransactionModel> {
     const result = await this.client.rawQuery(CATEGORIZE_TRANSACTION, args);
     return result.categorizeTransaction;
   }
+
+  private parseSearchQuery(searchQuery: string): TransactionFilter {
+    const searchTerms = searchQuery.slice(0, MAX_SEARCH_QUERY_LENGTH).split(" ");
+
+    const filter: TransactionFilter = {
+      iban_likeAny: searchTerms,
+      name_likeAny: searchTerms,
+      operator: BaseOperator.Or,
+      purpose_likeAny: searchTerms,
+    };
+
+    const amountRegex = /^-?\d+([,.]\d{1,2})?$/;
```

Comment:
So, I want to match: `1234`, `-1234`, `1.23`, `12.1`, `1,23`, `12,34` etc. etc.

I don't want to match shit like `.10`, `10.`, `10,` but maybe I should? 🤔

Reply:
I would vote for obvious ones only, so no .10 💩